### ISSUE TITLE
Ensure Featured Tutorials are up-to-date, accurate, and conform to guidelines

### ISSUE THREAD

**Issue by bermudezmt** [pulumi/docs#1671]
### Problem description

Revisit Featured Tutorials:
- [ ] AWS
- [ ] Azure
- [ ] GCP
- [ ] Kubernetes

Relates to https://github.com/pulumi/docs/issues/1656

**Comment by joeduffy** [comment]
@danfhernandez I know you've been going through these lately. We'd love your feedback overall on what needs improving. Also, would love if you can pair review some this as Maria makes progress!

**Comment by danfhernandez** [comment]
@joeduffy yeah happy to help here!  Let me know what I can do @bermudezmt.  

I have general recommendations for docs but nothing general on the tutorials, just a few specifics that I've noted so far.

**Comment by bermudezmt** [comment]
@danfhernandez I updated the issue to include all the featured tutorials. Thanks again for helping with this!

**Comment by danfhernandez** [comment]
Started working on the Azure examples.  See sheet for comments and progress:
https://docs.google.com/spreadsheets/d/1CA6qzit2AZjDFHP22lkG_TKekar1x0ontTyCYcU5FrY/edit?usp=sharing

**Comment by danfhernandez** [comment]
Two of the azure examples aren't working for me: 
1. [CosmosDB and Functions](https://www.pulumi.com/docs/tutorials/azure/azure-ts-serverless-url-shortener-global/)
- Failed at step 6.  Was not able to add a url shortener example.  Got Warning: Failed to open {"id":"pulumi","url":"https://pulumi.com"} 

2. [Azure App Service](https://www.pulumi.com/docs/tutorials/azure/azure-ts-appservice-springboot/)
- Fails with error: COPY failed: stat /var/lib/docker/tmp/docker-builder109089812/.classpath: no such file or directory at pulumi up

See more details here: https://docs.google.com/spreadsheets/d/1CA6qzit2AZjDFHP22lkG_TKekar1x0ontTyCYcU5FrY/edit?usp=sharing

@mikhailshilkov I see you're the author for these, can you check those to make sure it's not user error on my part?

**Comment by mikhailshilkov** [comment]
@danfhernandez 

Item (1) - Step 6 failed because `-D` option should be `-d`. Interestingly enough, this was [fixed](https://github.com/pulumi/examples/commit/384aaaea3c9a9619b83b6473b5d0ef0b279baf65#diff-730c0fb1210b49f70bee7e502d389b72) long time ago in the examples repo. It looks like we never update tutorials from the sources? I also opened [#524](https://github.com/pulumi/examples/pull/524) to improve the next step.

Item (2) - copying @praneetloke here as the original author

**Comment by lukehoban** [comment]
> It looks like we never update tutorials from the sources?

@joeduffy what was the originally intended process around this tool and keeping things up to date?  Can it be automated?

**Comment by lukehoban** [comment]
@danfhernandez @spara What's the overall status here.  Were the two issues Dan called out above the only two issues we identified?  Is there a finite set of work remaining?  Is there clear high priority work we are going to do to address the biggest opportunities?

**Comment by danfhernandez** [comment]
Those are the only blocking issues I found from the Azure tutorials but left more general feedback here: https://docs.google.com/spreadsheets/d/1CA6qzit2AZjDFHP22lkG_TKekar1x0ontTyCYcU5FrY/edit#gid=0.  This is something I'd like to wrap up.  I'll see if I can wrap this up by Friday.

**Comment by joeduffy** [comment]
> what was the originally intended process around this tool and keeping things up to date? Can it be automated?

No original thought here, sorry, other than "similar to our API docs update process" and "yes, it would be great to be automated" (assuming we want to keep this system here overall).
